### DISCUSSION TITLE
"Click the orange cursor to resume" for sentakki

### DISCUSSION THREAD

**Discussion by ziv-vy** [LumpBloom7/sentakki#629] [Ideas]
Feels way better than a countdown.

**Comment by LumpBloom7** [comment]
I disagree, because the cursor is not supposed to be the primary input method.

(the cursor resume overlay doesnt fit well with touchscreen)

**Reply by ziv-vy** [reply]
It would only pop up for mouse/GTablet users, and not for mobile users - just like osu!std.

**Comment by ziv-vy** [comment]
Superseded by #832
